dev_requirements.txt

```diff
@@ -0,0 +1,13 @@
+-r requirements.txt
```

Comment:
I would recommend dropping this extra file and just using the extra_requires options in `setup.cfg`. Essentially, just install within the config with `pip install -e .[tests]`. You can use the same for RTD (see [NiMARE's example](https://github.com/neurostuff/NiMARE/blob/master/.readthedocs.yml)).

Reply:
That would also mean dropping `requirements.txt` and changing your checksums from `requirements.txt` + `dev_requirements.txt` to `setup.cfg`.

Reply:
Thans @tsalo ! Trying to make that work now. Help appreciated!